tooling/minitest/src/pass/concurrency.rs

```diff
@@ -16,7 +17,7 @@ fn arbitrary_order() {
         );
         let b2 = block!(return_());
 
-        function(Ret::No, 0, &[], &[b0, b1, b2])
+        function(Ret::Yes, 1, &locals, &[b0, b1, b2])
```

Comment:
They would still be allowed to not have a return place, right? Why is this changed?

Reply:
Ah right.